recipes/nasa-smap-sss/meta.yaml

```diff
@@ -0,0 +1,21 @@
+title: "NASA SMAP Sea Surface Salinity (SSS)"
+description: "Analysis-ready Zarr datasets derived from NASA SMAP Sea Surface Salinity (SSS) NetCDF"
```

Comment:
I think it might be good to add that these are L3 (Level 3 gridded) data (there are also more complicated `swath` products). I don't think we need the NetCDF in the description?